src/python/grpcio_tests/tests/interop/server_ssl_cert_reload_test/_test.py

```diff
@@ -0,0 +1,363 @@
+# Copyright 2017 gRPC authors.
```

Comment:
Why locate this test in tests/interop rather than tests/unit? tests/interop is a bit of a special directory because everything there should have something to do with [the gRPC Interop Test Cases](https://github.com/grpc/grpc/blob/master/doc/interop-test-descriptions.md).

Reply:
will fix... it was just that it wasn't obvious to me where to put these tests. in the other PRs i had solicited feedback on a few possible locations but didn't get any feedback yet, so i just went with interop in the mean time

Reply:
done. moved to `unit`